Jamfile

```diff
@@ -396,7 +396,7 @@ rule openssl-lib-path ( properties * )
 		# on macOS, default to pick up openssl from the homebrew installation
 		# brew install openssl
 		# homebrew on M1 Macs install to /opt/homebrew
-		OPENSSL_LIB = /opt/homebrew/opt/openssl/lib /usr/local/opt/openssl/lib ;
+		OPENSSL_LIB = /opt/homebrew/opt/openssl@1.1/lib /opt/homebrew/opt/openssl/lib /usr/local/opt/openssl/lib ;
```

Comment:
I would prefer you put version specific paths in the workflow file rather than here.
you can specify this path on the `b2` command line by adding: `openssl-lib=/opt/homebrew/opt/openssl@1.1/lib`

Reply:
Will try this.... My concern is that the `cibuildwheels` wraps and obscures the invocation of `b2`.

Reply:
`cibuildwheels` seems to do the Python wrappers and `libtorrent-rasterbar` under openssl@3 ("brew install openssl" with no @<version>). The include & lib for openssl@3 are apparently found under `/opt/homebrew/opt/openssl`.